guider/guider.py

```diff
@@ -25690,8 +25690,12 @@ def runProcTop(self):
                     # print system status #
                     self.printSystemStat()
 
-                # report system status #
-                self.reportSystemStat()
+                if not SystemManager.elasticEnable:
```

Comment:
What do you think about changing it like this?
Because I think new visualization options will be added more :)

`if SystemManager.elasticEnable:
   self.reportSystemStatElastic()
else:
   self.reportSystemStat()
`

Reply:
I agree that